aicoe/add-image-to-datahub.md

```diff
@@ -1,10 +1,13 @@
 # Import Container Image to Internal Datahub:
 
-1. Create a new ImageStream template, you can use [this template](https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub).
-2. Fork this repo https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub.
-3. Add a new ImageStream to the [bases/custom-images/jupyterhub-custom-images.yaml](https://gitlab.cee.redhat.com/data-hub/dh-jupyterhub/blob/master/bases/custom-images/jupyterhub-custom-images.yaml).
-4. And create a Merge Request to publish these changes to the internal DataHub instance.
+* Create a new ImageStream template, you can use https://github.com/AICoE/s2i-custom-notebook#how-to-add-the-image-to-open-data-hubjupyterhub[this template].
```

Comment:
why is the link referencing format changed?

Reply:
My bad @harshad16, fixed that now!